website/docs/r/monitor_scheduled_query_rules_alert_v2.html.markdown

```diff
@@ -0,0 +1,191 @@
+  window_size          = "PT10M"
+  scopes               = [azurerm_application_insights.example.id]
+  severity             = 4
+  criteria {
+    query            = <<-QUERY
+      requests
+        | summarize CountByCountry=count() by client_CountryOrRegion
+      QUERY
+    time_aggregation = "Maximum"
+    threshold        = 17.5
+    operator         = "LessThan"
+
+    resource_id_column    = "client_CountryOrRegion"
+    metric_measure_column = "CountByCountry"
+    dimension {
+      name     = "client_CountryOrRegion"
+      operator = "Exclude"
+      values   = ["123"]
+    }
+    failing_periods {
+      min_failing_periods_to_alert = 1
+      number_of_evaluation_periods = 1
+    }
+  }
+
+  auto_mitigation_enabled                   = true
+  check_workspace_alerts_storage_configured = false
+  description                               = "example sqr"
+  display_name                              = "example-sqr"
+  enabled                                   = true
+  override_query_time_range                 = "PT1H"
+  skip_query_validation                     = true
+  action {
+    action_groups = [azurerm_monitor_action_group.example.id]
+    custom_properties = {
+      key  = "value"
+      key2 = "value2"
+    }
+  }
+
+  tags = {
+    key  = "value"
+    key2 = "value2"
+  }
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Specifies the name which should be used for this Monitor Scheduled Query Rule. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) Specifies the name of the Resource Group where the Monitor Scheduled Query Rule should exist. Changing this forces a new resource to be created.
+
+* `location` - (Required) Specifies the Azure Region where the Monitor Scheduled Query Rule should exist. Changing this forces a new resource to be created.
+
+* `criteria` - (Required) A `criteria` block as defined below.
+
+* `evaluation_frequency` - (Required) How often the scheduled query rule is evaluated, represented in ISO 8601 duration format. 
+
+* `scopes` - (Required) Specifies the list of resource ids that this scheduled query rule is scoped to. Changing this forces a new resource to be created.
+
+* `severity` - (Required) Severity of the alert. Should be an integer between 0 and 4. Value of 0 is severest. 
+
+* `window_size` - (Required) Specifies the period of time in ISO 8601 duration format on which the Scheduled Query Rule will be executed (bin size).
+
+* `action` - (Optional) An `action` block as defined below.
+
+* `auto_mitigation_enabled` - (Optional) Specifies the flag that indicates whether the alert should be automatically resolved or not. Value should be true or false. The default is false.
+
+* `check_workspace_alerts_storage_configured` - (Optional) Specifies the flag which indicates whether this scheduled query rule should be stored in the customer's storage. Value should be true or false. The default is false. 
```

Comment:
the property name indicates it checks something but the description states it does something?
```suggestion
* `check_if workspace_alerts_storage_is_configured` - (Optional) Specifies the flag which indicates whether this scheduled query rule check if storage is configured. Value should be true or false. The default is false. 
```
or
```suggestion
* `workspace_alerts_storage_enabled` - (Optional) Specifies the flag which indicates whether this scheduled query rule check if storage is configured. Value should be true or false. The default is false. 
```

Reply:
I think `workspace_alerts_storage_enabled` is better : )